rudder-core/src/main/scala/com/normation/rudder/domain/policies/Tags.scala

```diff
@@ -0,0 +1,90 @@
+/*
+*************************************************************************************
+* Copyright 2016 Normation SAS
+*************************************************************************************
+*
+* This file is part of Rudder.
+*
+* Rudder is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* In accordance with the terms of section 7 (7. Additional Terms.) of
+* the GNU General Public License version 3, the copyright holders add
+* the following Additional permissions:
+* Notwithstanding to the terms of section 5 (5. Conveying Modified Source
+* Versions) and 6 (6. Conveying Non-Source Forms.) of the GNU General
+* Public License version 3, when you create a Related Module, this
+* Related Module is not considered as a part of the work and may be
+* distributed under the license agreement of your choice.
+* A "Related Module" means a set of sources files including their
+* documentation that, without modification of the Source Code, enables
+* supplementary functions or services in addition to those offered by
+* the Software.
+*
+* Rudder is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
+
+*
+*************************************************************************************
+*/
+package com.normation.rudder.domain.policies
+
+import net.liftweb.common._
+import net.liftweb.util.Helpers.tryo
+import com.normation.utils.Control._
+
+/**
+ * Tags that apply on Rules and Directives
+ * We do not warranty unicity of tags name, only of tagsname + tagvalue
+ */
+final case class TagName( name : String )
+
+final case class TagValue( value : String )
+
+final case class Tag( tagName : TagName, tagValue : TagValue )
+
+/**
+ * We can have multiple Tags with same name - unicity is really on TagName + TagValue
+ */
+final case class Tags(
```

Comment:
I don't know what would be the advantage. It is more verbose, and the only possible advantage might be that one day Tags would be different, and so it wouild encapsulate change; but this is dubious
Should I drop it ?